extras/mmu/mmu.py

```diff
@@ -4285,8 +4285,23 @@ def _unload_bowden(self, length=None):
                     # Encoder based validation test
                     if self._can_use_encoder() and delta >= self.bowden_allowable_unload_delta and not self.calibrating:
                         ratio = 0.
-                        # Only a warning because _unload_gate() will deal with it
-                        self.log_info("Warning: Excess slippage was detected in bowden tube unload. Gear moved %.1fmm, Encoder measured %.1fmm" % (length, length - delta))
+                        # Check and attempt correction if bowden_apply_correction is enabled
+                        if self.bowden_apply_correction:
+                            self.log_debug("Checking if correction move is needed on unload bowden")
+                            for i in range(2):
+                                if delta >= self.bowden_allowable_unload_delta:
+                                    msg = "Correction unload move #%d from bowden" % (i + 1)
+                                    # Move "backwards" by delta since we still haven't offloaded enough
+                                    _, _, _, d = self.trace_filament_move(msg, -delta, track=True)
```

Comment:
Rather than create a temporary variable, use `delta`
```suggestion
                                    _, _, _, delta = self.trace_filament_move(msg, -delta, track=True)
```